dip-README.md

```diff
@@ -0,0 +1,36 @@
+# ChatDIP
+Welcome to the Digital Power AI chatbot solution based on this [Azure sample repository](https://github.com/Azure-Samples/azure-search-openai-demo).
+
+Since the authors of the Azure sample project have already documented almost all you need to know, this README is meant to navigate quickly to the relevant parts.
+
+## Directory structure
+The project constists of the following directories:
+- `/app` contains the Python (Quart) backend and the Typescript (React) frontend.
+- `/assets` contains images used in the app.
+- `/data` contains the files that are used to build the knowledge base. This is the data that the chatbot uses to answer questions!
+- `/docs` contains markdown files with documentation.
+- `/infra` contains the Bicep files defining the required Azure resources.
+- `/scripts` contains Shell & PowerShell scripts for adhoc functionalities.
+- `/tests` contains Pytest tests
+
+## Get started
+To collaborate on this project in an organised way we follow the [Github flow](https://docs.github.com/en/get-started/using-github/github-flow) principles. This effectively means that all changes require a separate branch and a pull request to the `main` branch **with** reviewers to process them.
+
+### Changing the data
+The easiest way to change the underlying data that the chatbot uses, is to replace the files in the `/data` folder. After committing and pushing the code the to remote `main` branch, a "deploy" pipeline will upload the data to the Azure environment.
```

Comment:
I think we should add here that this does not delete data that is already indexed.